council/llm/anthropic_messages_llm.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import annotations
+
+from typing import Sequence, List, Iterable, Literal
+
+from anthropic import Anthropic
+from anthropic._types import NOT_GIVEN
+from anthropic.types import MessageParam
+
+from council.llm import (
+    LLMMessage,
+    LLMMessageRole,
+    AnthropicLLMConfiguration,
+)
+from council.llm.anthropic import AnthropicAPIClientWrapper
+
+
+class AnthropicMessagesLLM(AnthropicAPIClientWrapper):
+    """
+    Implementation for an Anthropic LLM.
```

Comment:
Maybe add note like: To be used with LLMs using the `messages` endpoint", like `Claude-2.1` and `Claude-3`